stellarphot/settings/models.py

```diff
@@ -252,23 +252,27 @@ class Camera(BaseModelWithTableRep):
         NonEmptyStr,
         Field(
             description="Name of the camera",
-            examples=["SBIG ST-8300M", "ZWO ASI1600"],
+            examples=["SBIG FakeCam", "ZWO NadaCam", "CG16m"],
```

Comment:
Ah, this may have been my issue in testing the settings notebook for PR#294, I bet you I had inconsistent units.